content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -55,22 +55,65 @@ The underlying SQL looks like this:
 ```sql
 CREATE TABLE "Category" (
     id SERIAL PRIMARY KEY
+    name TEXT NOT NULL
 );
+
 CREATE TABLE "Post" (
     id SERIAL PRIMARY KEY
+    title TEXT NOT NULL
 );
--- Relation table + indexes -------------------------------------------------------
+
+-- Relation table  -------------------------------------------------------
+CREATE TABLE "CategoriesOnPosts" (
+    "categoryId" INTEGER NOT NULL,
+    "postId" INTEGER NOT NULL,
+    "assignedBy" TEXT NOT NULL,    
+    "assignedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY ("categoryId") REFERENCES "Category"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    FOREIGN KEY ("postId") REFERENCES "Post"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    CONSTRAINT "CategoriesOnPosts_pkey" PRIMARY KEY ("postId", "categoryId")
+);
```

Comment:
I agree with the capitalization changes here.

But why the change from Unique index to PK?
Why the addition of the referential actions?
(Both might be justified, I just want to ahve the explicit reason to understand.)